apps/ui/src/components/ui/nav-link.tsx

```diff
@@ -34,13 +34,10 @@ export const NavLink = ({
 }: NavLinkProps) => {
   const pathname = usePathname();
   const isActive = exact ? pathname === href : pathname.startsWith(href);
-
-  variant = "ghost";
   return (
     <Link
       className={cn(
         buttonVariants({ variant, size, className }),
-        "pb-4",
```

Comment:
this `pb-4` class was adding extra padding below the text of links. I removed it because it was messing up the layout of content provided to links in stories.